appveyor.yml

```
@@ -0,0 +1,33 @@
+environment:
+  JULIA_TZ_VERSION: "2016j"
+
+  matrix:
+  - julia_version: 1.0
+  - julia_version: 1.1
+  - julia_version: 1.2
+  - julia_version: 1.3
+  - julia_version: latest
```

This apparently works. Let's just update to that right away
```suggestion
  - julia_version: nightly
```